vaadin-radio-button-flow-parent/vaadin-radio-button-flow/src/main/java/com/vaadin/flow/component/radiobutton/RadioButtonGroup.java

```diff
@@ -245,6 +245,7 @@ protected void onDetach(DetachEvent detachEvent) {
         	dataProviderListenerRegistration = null;
         }
         super.onDetach(detachEvent);
+        reset();
```

Comment:
Using `reset()` seems to also `clear()` the component's value. So on detach, the RBG would have its value set to `null`.

Reply:
You are right, probably I will also add test for default `detach-attach` without `PolymerTemplate` to test this usecase.

Reply:
Added test to cover this usecase as well.

Reply:
![Screenshot 2020-10-22 at 10 03 49](https://user-images.githubusercontent.com/1222264/96836750-14913f80-144e-11eb-8e54-b0d05e04b876.png)

Not sure why the existing buttons/radio group weren't re-used for the new test. Nothing blocking, but generally the test pages should also be easy to understand.